tf2onnx/optimizer/transpose_optimizer.py

```diff
@@ -79,6 +79,31 @@ def pre_optimize_action(self):
             self._g.topological_sort(ops)
 
     def post_optimize_action(self):
+        nodes = self.nodes
+        # if channel==1 or height==width==1, replace transpose with reshape
+        for op in nodes:
+            if op.type == "Transpose":
+                input_shape = self._g.get_shape(op.input[0])
+                new_shape = []
+                if is_nchw_transpose(op) and (input_shape[3] == 1 or (input_shape[1] == 1 and input_shape[2] == 1)):
+                    new_shape = [input_shape[0], input_shape[3], input_shape[1], input_shape[2]]

```

Comment:
```if is_nchw_transpose``` and than transpose to 0,3,1,2 (aka NHWC_TO_NCHW) ... this is correct? 
A little comment would help.

Reply:
Added